### DISCUSSION TITLE
shading error or my pc?

### DISCUSSION THREAD

**Discussion by idalx** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1817] [General]
![Captura de Tela (622)](https://github.com/user-attachments/assets/c641197f-7b13-4d09-9cc4-3096e96989dd)
I was playing normally until I found a crashed ship and came across this texture bug, what about my PC? or some kind of graphical error in the mod because the other mods are normal

**Comment by DerKaiser2023** [comment]
What mods do you have installed?

**Reply by idalx** [reply]
Better-Ping-Display-Med-Forge-1.7.10
BuildCraft-Mod-1.7.10 jar
CodeChicken Core-1.7.10-1.0.7.48-universaljar
CoreTweaks-1.7.10-0.3.3.2.jar
Damage-Indicators-Mod-1.7.10.jar
Galacticraft-Mod-1.7.10 jar 
Galacticraft-Planets Mod 17.10
HBM NTM 1.0.27 X5335.jar
Inventory-Tweaks-Med-1.7.10.jar
Iron-Chests-Mod-1.7.10jar
iron-Furnaces-Mod-1.7.10.jar
MicdoodleCore-1.7.10 jar
NotEnoughitems-1.7.10-1.0.5.120-universaljar 
OptiFine-1.7.10 HD UE7 jar
Reis-Minimap-Mod-1.7.10.jar
CodeChickenLib-1.7.10-1.1.3.141-universal.jar

**Reply by burningwater202** [reply]
Try removing Optifine and see if that fixes it

**Reply by idalx** [reply]
it worked thank you
